scripts/index/publish_artifacts.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import os
+import boto3
```

Comment:
How are we getting `boto3`?

Reply:
I'm not sure, but all of Bilal's other python scripts use this too without any sort of obvious virtualenv pulling it in. I assume the CI machines have it builtin.